scripts/quickstart.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+
+setup_redhat_based_system() {
+
+  echo "setting up the needed system level tools"
+  pkcon install -y \
+      python-pip gcc postgresql-devel \
+      libxml2-devel libxslt-devel \
+      zeromq3-devel libcurl-devel \
+      redhat-rpm-config gcc-c++ \
+      openssl-devel libffi-devel
+
+  echo "user install of the latest pip/virtualenv to avoid old distro packages"
+  pip install -U -q --user pip virtualenv
+}
+
+
+patch_activate() {
+  if grep -q  PYTHONDONTWRITEBYTECODE ./.cfme_tests/bin/activate
+  then
+    echo "Activate Script already patched"
+  else
+    echo "Patching Activate Script"
+    cat >> ./.cfme_tests/bin/activate <<EOF;
+
+    export PYTHONDONTWRITEBYTECODE=yes
+    if [[ $PYTHONPATH ]]
+    then
+      PYTHONPATH+=":`pwd`"
```

Comment:
i think it shouldn't matter, i followed the suggestions i received in `#bash` when reeducating myself about the pleasures of bashing my head